lib/contracts/contributor.js

```diff
@@ -1,5 +1,6 @@
 const Record = require('./record');
 const ContributorSerializer = require('../serializers/contributor');
+const KreditsFormatter = require('../utils/kredits-formatter');
```

Comment:
As this is not a class, but just one formatting function, I'd calls the file `format_kredits` and import it as `formatKredits()`.

Reply:
I think the convention is a dash for filenames in JS.
👍 `formatKredits()`